extensions/adobe/experience/customerJourneyManagement/message-delivery-feedback.schema.json

```diff
@@ -0,0 +1,97 @@
+  ],
+  "$id": "https://ns.adobe.com/xdm/experience/customerJourneyManagement/message-delivery-feedback",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe CJM ExperienceEvent - Delivery-Feedback Details",
+  "description": "Provide details for the Delivery feedback for a single message in the MessageExecution.",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+
+  "definitions": {
+    "messageDeliveryfeedback": {
+      "properties": {
+        "xdm:feedbackStatus": {
+          "title": "Feedback Status",
+          "type": "string",
+          "description": "Provide status of a delivery attempt.",
+          "enum": ["error", "exclude", "sent", "bounce", "delay"],
+          "meta:enum": {
+            "error": "Mesage processing failed.",
+            "exclude": "Messageprofile is filtered out for sending the message",
+            "sent": "Messages such as Email, SMS, Push that have been dispatched to a person's account, address, or device.",
+            "bounce": "Message failed to deliver, and no longer will be retried.",
+            "delay": "message failed to deliver, but will be retry after next schedule time."
+          }
+        },
+        "xdm:excludeReason": {
+          "title": "Failure Code",
+          "type": "object",
+          "properties": {
+            "xdm:excludeCode": {
+              "title": "Exclude Code",
+              "type": "string",
+              "description": "Provide top level exclusion reason, like typology rule, mandatory parameter missing. ",
+              "meta:enum": {
+              }
+            },
+            "xdm:excludeSubCode": {
+              "title": "Exclude SubCode",
+              "type": "string",
+              "description": "provide exclusion subcode, like exact typology rule-id."
+            }
+          }
+        },
+        "xdm:messageFailure": {
+          "title": "Message Failure",
+          "type": "object",
+          "properties": {
+            "xdm:failureReason": {
+              "title": "Failure Reason",
+              "type": "string",
+              "description": "Failure reason for a failed delivery attempt."
+            },
+            "xdm:failureCode": {
+              "title": "Failure Code",
+              "type": "string",
+              "description": "Failure code for a failed delivery attempt."
+            },
+            "xdm:failureType": {
+              "title": "Failure Type",
+              "type": "string",
+              "description": "Classifies whether it is a soft bounce or hard bounce.",
+              "meta:enum": {
+                "soft": "The reason for failed delivery is temprorary.",
+                "hard": "The reason for failed delivery is permanent."
+              }
+            }
+          }
+        },
+        "xdm:retryCount": {
```

Comment:
This field can be removed, as there is no reporting/segmentation use case for this.